src/components/ReadArticle/ReadArticle.spec.js

```diff
@@ -0,0 +1,125 @@
+import { ReadArticle, mapStateToProps } from './ReadArticle';
+import ArticleAuthor from './ArticleAuthor';
+import ArticleTags from './ArticleTags';
+import ArticleOptions from './ArticleOptions';
+
+describe('Read Article component test', () => {
+  it('mounts the ReadArticle Component and renders DOM elements', () => {
+    const props = {
+      article: {},
+      history: {
+        push: jest.fn()
+      },
+      match: {
+        params: '/article'
+      },
+      getArticle: jest.fn(),
+      loading: false
+    };
+    const wrapper = shallow(<ReadArticle {...props} />);
+    expect(wrapper.find('main')).toHaveLength(1);
+    expect(wrapper.find('section')).toHaveLength(3);
+  });
+
+  it('mounts ReadArticle when loading is false', () => {
+    const props = {
+      article: {
+        title: 'rambo test article',
+        Author: {
+          firstName: 'Rambo dev',
+          lastName: 'Rambo dev',
+          userName: 'Rambo dev',
+          avatarUrl: 'http:www.url.com'
+        }
+      },
+      match: {
+        params: '/article'
+      },
+      getArticle: jest.fn(),
+      loading: true
```

Comment:
@LukasChiama Your test title shows you asserting for when loading is false. However, your loading prop is set to true. I guess missed that part.

Reply:
Thanks @abdulfatai360. It was definitely an oversight. I'll fix it immediately